pyro/infer/autoguide/guides.py

```diff
@@ -874,27 +875,31 @@ def _setup_prototype(self, *args, **kwargs):
         super()._setup_prototype(*args, **kwargs)
         # Initialize guide params
         self.loc = nn.Parameter(self._init_loc())
+        self.scale = PyroParam(
+            torch.full_like(self.loc, self._init_scale), self.scale_constraint
+        )
         self.scale_tril = PyroParam(
-            eye_like(self.loc, self.latent_dim) * self._init_scale,
-            self.scale_tril_constraint,
+            eye_like(self.loc, self.latent_dim), self.scale_tril_constraint
         )
 
     def get_base_dist(self):
         return dist.Normal(
-            torch.zeros_like(self.loc), torch.zeros_like(self.loc)
+            torch.zeros_like(self.loc), torch.ones_like(self.loc)
```

Comment:
Hmm, what a bug!? It is surprised to me that we didn't catch this earlier.

Reply:
well, apparently nobody uses `AutoMultivariateNormal` with `NeuTraReparam`.  My motivation for this PR is to write a tutorial about autoguides, so hopefully they'll get more exposure